web/src/index.js

```diff
@@ -1 +1,69 @@
-console.log('Hello world !');
+var body = document.body;
+
+var formElt = document.createElement("form");
+var nameInputElt = document.createElement("input");
+var urlInputElt = document.createElement("input");
+var envoyerElt = document.createElement("input");
+
+envoyerElt.type = "submit";
+
+formElt.appendChild(nameInputElt);
+formElt.appendChild(urlInputElt);
+formElt.appendChild(envoyerElt);
+
+body.appendChild(formElt);
+
+formElt.addEventListener("submit", function (e) {
+    //e.preventDefault();
+    var newItem = new FormData();
```

Comment:
Try to use `let` and `const` keywords instead of `var`.
Most of the time you can use `const`, which declares a variable in current block and prevent it from beeing reassigned, IE it gives stronger guarantees.

See [here](https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75) for more informations.